Dockerfile.debug

```diff
@@ -12,12 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.13.15 as builder
+FROM golang:1.19 as builder
 WORKDIR /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver
 ADD . .
 
-RUN CGO_ENABLED=0 go get -ldflags "-s -w -extldflags '-static'" github.com/go-delve/delve/cmd/dlv
-RUN GCE_PD_CSI_DEBUG=1 make gce-pd-driver
+ARG STAGINGVERSION
+ARG TARGETPLATFORM
+
+RUN CGO_ENABLED=0 go install -ldflags "-s -w -extldflags '-static'" github.com/go-delve/delve/cmd/dlv@latest
```

Comment:
I think that because this is a dev image it's ok if it's latest i.e. there's nothing in CI that would break